eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/Channel.scala

```diff
@@ -1110,6 +1110,11 @@ class Channel(val nodeParams: NodeParams, val wallet: OnChainChannelFunder, val
           log.info("channelId={} was confirmed at blockHeight={} txIndex={} with a previous funding txid={}", d.channelId, w.blockHeight, w.txIndex, w.tx.txid)
           watchFundingTx(commitments1)
           context.system.eventStream.publish(TransactionConfirmed(d.channelId, remoteNodeId, w.tx))
+          val otherFundingTxs = d.fundingTx match {
+            case Some(DualFundedUnconfirmedFundingTx(fundingTx)) => fundingTx +: d.alternativeCommitments.filter(_.commitments.commitInput.outPoint.txid != w.tx.txid).map(_.fundingTx)
+            case _ => d.alternativeCommitments.filter(_.commitments.commitInput.outPoint.txid != w.tx.txid).map(_.fundingTx)
+          }
```

Comment:
```suggestion
          val otherFundingTxs =
            d.fundingTx.toSeq.collect { case DualFundedUnconfirmedFundingTx(fundingTx) => fundingTx } ++
              d.alternativeCommitments.filterNot(_.commitments.commitInput.outPoint.txid == w.tx.txid).map(_.fundingTx)
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2390/commits/5eda6bd4fff10fd8ee0d68f5f20addb34901d343